### PR TITLE
Removing style ci requirement

### PR DESCRIPTION
**Pull request by BKorenek**
https://github.com/medology/stdcheck.com/issues/9490
Updating pull approve to not include style ci requirement

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -18,11 +18,7 @@ pullapprove_conditions:
     explanation: "Not ready for review"
   - condition: "state == 'open'"
     unmet_status: failure
-    explanation: "PR must be open to be reviewed"
-  - condition: "'continuous-integration/styleci/pr' in statuses.succeeded or 'Force Review' in labels"
-    unmet_status: pending
-    explanation: "StyleCI must pass before requesting review"
-  - condition: "'ci/circleci: test' in statuses.succeeded or 'Force Review' in labels"
+    explanation: "PR must be open to be reviewed"  - condition: "'ci/circleci: test' in statuses.succeeded or 'Force Review' in labels"
```
Comment (izazueta) [inline .pullapprove.yml:21]:
```suggestion
    explanation: "PR must be open to be reviewed"  
  - condition: "'ci/circleci: test' in statuses.succeeded or 'Force Review' in labels"
```

File: .pullapprove.yml [hunk 2]
```diff
@@ -18,10 +18,7 @@ pullapprove_conditions:
     explanation: "Not ready for review"
   - condition: "state == 'open'"
     unmet_status: failure
-    explanation: "PR must be open to be reviewed"
-  - condition: "'continuous-integration/styleci/pr' in statuses.succeeded or 'Force Review' in labels"
-    unmet_status: pending
-    explanation: "StyleCI must pass before requesting review"
+    explanation: "PR must be open to be reviewed"
```
Comment (Chekote) [inline .pullapprove.yml:21]: You added unnecessary whitespace to the end of this line.

Comment (Chekote) [reply .pullapprove.yml:21]: This comment wasn't addressed/

Comment (BKorenek) [reply .pullapprove.yml:21]: I accidently did this one instead of the other one, but I also don't see any white space on my end

Comment (BKorenek) [reply .pullapprove.yml:21]: never mind
